docker/Makefile

```diff
@@ -230,9 +224,6 @@ ub-18.04-irods-clients-4.2.12.$(TAG): irods_clients/ubuntu/Dockerfile ub-18.04-b
 	--build-arg DOCKER_IMAGE=$(subst .$(TAG),,$@) \
 	--build-arg DOCKER_TAG=$(TAG) \
 	--build-arg IRODS_VERSION=4.2.12 \
-	--build-arg BATON_VERSION=4.1.0 \
-	--build-arg HTSLIB_VERSION=1.17 \
-	--build-arg SAMTOOLS_VERSION=1.17 \
 	--label org.opencontainers.image.title="iRODS 4.2.12 clients, Ubuntu 18.04" \
 	--label org.opencontainers.image.source=$(git_url) \
 	--label org.opencontainers.image.revision=$(git_commit) \
```

Comment:
That's not the direction of travel that I'm looking for wrt build args. They are collected in the Makefile so that they are all visible in one place, for ease of maintenance. That's based on the experience of making a lot of managing a lot of version updates.

That they still have default values in the Dockerfiles because I've not fully migrated them upwards - I'm intending to remove the defaults, make the build-args mandatory and expose them as Makefile arguments.

This change also leaves the iRODS client version being handled differently from the other clients (that still has a build-arg). The should all behave the same way.

If the issue is one of clarity on how to update versions, the best fix would be to add a sentence to the README explaining where to make the changes.

Reply:
VERSIONs back in with rebase (force  21ca1db to 4f41946 )